pyaedt/modules/MeshIcepak.py

```diff
@@ -541,8 +541,8 @@ class MeshRegionCommon(object):
             Name of the mesh region.
         manual_settings : bool
             Whether to use manual settings or automatic ones.
```

Comment:
How does one know how to set this parameter? I'd recommend making it clear by rewriting something like this: Whether to use manual settings. If ``False``, automatic settings are used.